setup.py

```diff
@@ -52,19 +67,45 @@ def main():
     class SetupStepFailedError(ValueError):
         pass
 
-    def getInputPath(prompt):
-        import shlex
-
+    def getOutputPath(prompt, default):
+        prompt_cli = prompt + f' ({default=}): '
         while True:
-            path = ''
-            while len(path) == 0:
-                path = input(prompt + ' ')
+            if gui_mode:
+                filename = filedialog.asksaveasfilename(confirmoverwrite=False, initialfile=default, title=prompt, defaultextension=".bin", filetypes=(("BIN file", ""), ))
```

Comment:
Can this throw?

Reply:
The *asfilename methods only retrieve a path from the filesystem and it's up to developer to do stuff with it.
Only place where I could think of this throwing now (with try-except around the root Tk init), is a weird customized linux install where tkinter can initialize but native filedialogs aren't available.

Reply:
Okay, sounds good.